docs/releases/1.11.txt

```diff
@@ -402,6 +402,8 @@ Tests
 * Added support for :meth:`python:unittest.TestCase.subTest`’s when using the
   :option:`test --parallel` option.
 
+* Tests now run checks at the start of the test run.
```

Comment:
Could you explain that it's `DiscoverRunner` that does this. Perhaps the docs for `DiscoverRunner` should also be updated.